missions/ME_Austria_Missions.txt

```diff
@@ -1,1674 +1,6981 @@
-			capital_scope = {
-				area = {
-					limit = { owned_by = ROOT }
-					add_province_modifier = {
-						name = "growth_of_capital"
-						duration = 9125 #25 years
+			custom_tooltip = me_hab_salt_bavarian_provinces
+			ME_distribute_development = { 
+				type = production
+				amount = 10
+				limit = " 
+					OR = {
+						area = lower_bavaria_area
+						area = upper_bavaria_area
+						area = east_bavaria_area
 					}
-				}
+					trade_goods = salt
+				"
 			}
 		}
 	}
 	
-	austria_build_universities = {
-		icon = me_mission_generic_old_university
-		position = 12
-		required_missions = { austria_develop_capital }
+	me_hab_integrate_bavaria = {
+		icon = bavarian_cultural_dominance
+		required_missions = {
+			me_hab_bavaria
+		}
+		position = 3
 		trigger = {
-			university = 5
+			lower_bavaria_area = {
+				type = all
+				owned_by = ROOT
+				is_state_core = ROOT
+				NOT = {
+					nationalism = 1
+				}
+			}
+			upper_bavaria_area = {
+				type = all
+				owned_by = ROOT
+				is_state_core = ROOT
+				NOT = {
+					nationalism = 1
+				}
+			}
+			east_bavaria_area = {
+				type = all
+				owned_by = ROOT
+				is_state_core = ROOT
+				NOT = {
+					nationalism = 1
+				}
+			}
 		}
 		effect = {
-			if = {
-				limit = {
-					has_dlc = "Rule Britannia"
-				}
-				change_innovativeness = 5
+			65 = {
+				center_of_trade = 2
 			}
-			else = {
-				add_adm_power = 25
-				add_dip_power = 25
-				add_mil_power = 25
+			lower_bavaria_area = {
+				add_prosperity = 100
+			}
+			upper_bavaria_area = {
+				add_prosperity = 100
+			}
+			east_bavaria_area = {
+				add_prosperity = 100
 			}
 		}
 	}
 }
 
-ME_austria_missions_4 = {
-	slot = 4
+me_hab_2_1 = {
+	slot = 2
 	generic = no
 	ai = yes
 	potential = {
-		AND = {
-			tag = HAB
-			has_country_flag = use_missions_expanded_missions_instead_of_paradox
-		}
+		tag = HAB
+		has_country_flag = use_missions_expanded_missions_instead_of_paradox
 	}
 	has_country_shield = yes
 	
-	austria_secure_bohemia = {
-		icon = me_mission_bohemia_flag
-		position = 2
-		required_missions = { austria_control_electors }
+	me_hab_venice = {
+		icon = expand_the_doges_palace
+		required_missions = {
+			me_hab_slovakia
+		}
+		position = 5
+		provinces_to_highlight = {
+			area = venetia_area
+			NOT = { country_or_non_sovereign_subject_holds = ROOT }
+		}
 		trigger = {
-			OR = {
-				AND = {
-					is_emperor = yes
-					BOH = {
-						preferred_emperor = ROOT
-						has_opinion = { who = ROOT value = 100 }
+			if = {
+				limit = {
+					exists = VEN
+				}
+				offensive_war_with = VEN
+				OR = {
+					112 = {
+						controlled_by = ROOT
+					}
+					war_score_against = {
+						who = VEN
+						value = 50
+					}
+				}
+			}
+			else = {
+				custom_trigger_tooltip = {
+					tooltip = me_own_entire_venetia_area_except_venice
+					venetia_area = {
+						type = all
+						OR = {
+							owned_by = ROOT
+							province_id = 112
+						}
 					}
-					marriage_with = BOH
 				}
```

Comment:
discrepancy (112 is not required, also should check for ownership in provinces to highlight)